src/Processors/Transforms/SquashingChunksTransform.h

```diff
@@ -28,4 +29,23 @@ class SquashingChunksTransform : public ExceptionKeepingTransform
     Chunk finish_chunk;
 };
 
+/// Doesn't care about propagating exceptions and thus doesn't throw LOGICAL_ERROR if the following transform closes its input port.
+class SimpleSquashingChunksTransform : public ISimpleTransform
```

Comment:
This mane violates the rule: *Never use the word* **simple** *in any name* :D
(ISimpleTransform is a bad name as well)